src/test/integration/controllers/attachments.spec.js

```diff
@@ -138,6 +138,10 @@ describe('Attachments controller', () => {
             link: expect.any(String),
             size: 1524,
             category: null,
+            availability: {
+              status: 'open',
+              date: null
+            },
```

Comment:
Maybe also reuse it here? What do you think about adding `common.js` in test `folder` and place it there?

Reply:
I think it's unnecessary for this case because we have a lot of repeating in our tests. So it would be better to create an issue for creating a new file in our architecture where we can move this code. Also, we can discuss this topic in the next meeting

Reply:
I believe creating `constants` file or folder can be a good idea